scroll-animations-1/Overview.bs

```diff
@@ -736,67 +736,79 @@ spec: cssom-view-1; type: dfn;
 
 	A set of animation keyframes can be attached
 	in reference to [=named timeline ranges=]
-	by adjusting the duration of the animation.
+	by restricting the to the specified timeline range.
+	Delays ('animation-start-delay'/'animation-end-delay')
+	are set within this restricted range.
+	If a duration ('animation-duration') is given,
+	it measures from the range-restricted start,
+	and further restricts the range-restricted end.
 
 	<pre class="propdef shorthand">
-		Name: animation-delay, animation-range
-		Value: [ <<'animation-delay-start'>> <<'animation-delay-end'>>? | <<timeline-range-name>> ]#
+		Name: animation-range
+		Value: [ <<'animation-range-start'>> <<'animation-range-end'>>? | <<timeline-range-name>> ]#
 	</pre>
 
-	The 'animation-delay' and 'animation-range' properties
-	are both [=shorthand property|shorthands=]
-	that set 'animation-delay-start' and 'animation-delay-end'
+	The 'animation-range' property is a [=shorthand property|shorthand=]
+	that sets 'animation-range-start' and 'animation-range-end'
 	together in a single declaration.
-	If the <<'animation-delay-end'>> value is omitted,
+	If the <<'animation-range-end'>> value is omitted,
 	it is set to zero.
 	If a <<timeline-range-name>> alone is specified,
-	'animation-delay-start' is set to that name plus ''0%''
-	and 'animation-delay-end' is set to that name plus ''100%''.
+	'animation-range-start' is set to that name plus ''0%''
+	and 'animation-range-end' is set to that name plus ''100%''.
 
 	<pre class="propdef">
-		Name: animation-delay-start
-		Value: [ <<time>> | <<timeline-range-name>> <<percentage>> ]#
-		Initial: 0s
+		Name: animation-range-start
+		Value: [ normal | <<timeline-range-name>> <<percentage>> ]#
+		Initial: normal
 		Applies to: all elements
 		Inherited: no
 		Percentages: relative to the specified [=named timeline range=]
-		Computed value: list, each item either a duration or a timeline range and progress percentage
+		Computed value: list, each item either the keyword ''normal'' or a timeline range and progress percentage
 		Animatable: no
 	</pre>
 
-	Defines a delay between the start of the animation
-	and when it begins executing
-	(i.e. when keyframes attached to 0% progress are mapped).
+	Shifts the start of the animation
+	(i.e. when keyframes attached to 0% progress are mapped)
+	to the specified position on the timeline.
 
-	<dl dfn-for="animation-delay-start, animation-delay-end" dfn-type=value>
-		<dt><dfn><<time>></dfn>
+	<dl dfn-for="animation-range-start" dfn-type=value>
+		<dt><dfn>auto</dfn>
 		<dd>
-			Defines the delay as a duration. See [[CSS-ANIMATIONS-1]].
+			The start of the animation attaches to the start of the timeline.
```

Comment:
Since auto is the default value it will have to have the current behavior for regular time based timelines. That is,

If the timeline is monotonically increasing:
- The start of the animation attaches to the current time of the timeline.
- Otherwise, the start of the animation attaches to the start of the timeline.

This is essentially restating the behavior specified in steps 5-7 of playing an animation:
https://drafts.csswg.org/web-animations-1/#playing-an-animation-section

Reply:
I switched it to “The [=end time=] of the animation is determined as normal.”